assets/docs/callmemehdi/models/AraBERT/1.md

```diff
@@ -0,0 +1,44 @@
+# Module callmemehdi/arabert/1
+
+AraBERT is an Arabic pretrained lanaguage model based on Google's BERT architechture.
+
+<!-- asset-path: https://gsoctfarabert.web.app/arabert.tar.gz -->
+<!-- task: text-embedding -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
+<!-- license: custom -->
+
+## Overview
+
+These results are obtained from pre-training BERT specifically for the Arabic language in the pursuit of achieving the same success that BERT did for the English language.The performance of AraBERT is compared to multilingual BERT from Google and other state-of-the-art approaches
+
+## Performance
+
+Performance of AraBERT on Arabic downstreamtasks compared to mBERT
+Results:
+
+ Task | metric | mBERT | AraBERT
+------|--------|------ | -------
+SA (HARD)        | Acc.   | 95.7  | 96.1
+SA (ASTD)        | Acc.   | 80.1  | 96.5
+SA (ArsenTD-Lev) | Acc.   | 51.0  | 59.4
+SA (AJGT)        | Acc.   | 83.6  | 93.8
+SA (LABR)        | Acc.   | 83.0  | 86.7
+NER (ANERcorp) | macro-F1   | 78.4  | 81.9
+QA (ARCD) | macro-F1   | 61.3  | 62.7
+
```

Comment:
Can you add a section that shows how the model is loaded (eg. hub.load or hub.keraslayer), and how it is used (eg. what inputs and outputs look like). See https://tfhub.dev/tensorflow/small_bert/bert_en_uncased_L-4_H-512_A-8/2 for a good example.

Reply:
Also what kind of preprocessor needs to be used?

Reply:
For this, @MorganR , I was thinking to include it in the example, after the model is finished, that way I know the code exactly on how to load it. I can include the preprocessing part, though, and bring it up again in the example. What do you think?

Reply:
Sounds good, including preprocessor information in the example would be great.